src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/InnerApi/CandidateApi/Requests/PostCandidateAddressApiRequest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.FindAnApprenticeship.InnerApi.CandidateApi.Requests;
+public class PutCandidateAddressApiRequest : IPutApiRequest
+{
+    private readonly string _govUkIdentifier;
+    public object Data { get; set; }
+
+    public PutCandidateAddressApiRequest(string govIdentifier, PutCandidateAddressApiRequestData data)
+    {
+        _govUkIdentifier = govIdentifier;
+        Data = data;
+    }
+
+    public string PutUrl => $"api/addresses/{_govUkIdentifier}";
```

Comment:
i think the resource might be better described as:
```suggestion
    public string PutUrl => $"api/candidates/{_candidateId}/addresses/";
```
we should know the candidate id when doing the request